tests/test_api_call.py

```diff
@@ -3,45 +3,47 @@
 
 from api_call import *
 
-key = os.environ['API_TOKEN_REGULATIONS_GOV']
 base_url = 'https://api.data.gov:443/regulations/v3/documents.json?'
 
+
 @pytest.fixture
 def mock_req():
     with requests_mock.Mocker() as m:
         yield m
 
+
 def test_happy_path(mock_req):
     mock_req.get(add_api_key(base_url), status_code=200, text='{}')
-    assert call(add_api_key(base_url)) == {}
+    assert call(base_url).text == '{}'
 
 
 def test_exception_thrown(mock_req):
     mock_req.get(add_api_key(base_url),status_code=300)
     with pytest.raises(TemporaryException):
-        call(add_api_key(base_url))
+        call(base_url)
 
 
 def test_api_count_zero(mock_req):
         mock_req.get(add_api_key(base_url), status_code=429)
         with pytest.raises(ApiCountZeroException):
-            call(add_api_key(base_url))
+            call(base_url)
 
 
-def test_api_permanent_Exception(mock_req):
+def test_api_permanent_exception(mock_req):
         mock_req.get(add_api_key(base_url), status_code=404)
```

Comment:
I notice one commit changed " call(add_api_key(base_url))" to "call(base_url)". Should "add_api_key(base_url)" in all of these mocker method calls be changed to "base_url" as well?